packages/compass-web/src/index.tsx

```diff
@@ -152,6 +163,40 @@ const CompassWeb = ({
   const [connectionError, setConnectionError] = useState<any | null>(null);
   const dataService = useRef<DataService>();
 
+  const loggerProvider = useRef({
+    createLogger: (component = 'SANDBOX-LOGGER'): LoggerAndTelemetry => {
```

Comment:
As this is actually a sandbox logger and we do plan to eventually pass a real one from the mms side, I think this setup code should stay on the sandbox side so that we don't bundle this for now

Reply:
aah good point. I just hacked it early on then totally forgot I meant to move that.